### PR TITLE
Add ISerializable to WebException

### PR DESCRIPTION
**Pull request by tijoytom**
Fixes https://github.com/dotnet/corefx/issues/12138

### CODE REVIEW

**Comment by jkotas** [comment]
@tijoytom Could you please follow https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/contributing.md for writing your commit descriptions?

In particular: `DO give PRs short-but-descriptive names (e.g. "Improve code coverage for System.Console by 10%", not "Fix #1234")`

File: src/System.Net.Requests/src/System/Net/WebException.cs [hunk 1]
```diff
@@ -65,6 +70,16 @@ public WebResponse Response
             }
         }
 
+        void ISerializable.GetObjectData(SerializationInfo serializationInfo, StreamingContext streamingContext)
+        {
+            GetObjectData(serializationInfo, streamingContext);
+        }
```
Comment (stephentoub) [inline src/System.Net.Requests/src/System/Net/WebException.cs]: Do you know why it's not serializing at least ```_status```?  I see that code is commented out:
https://referencesource.microsoft.com/#System/net/System/Net/WebException.cs,163
It'd be interesting to know why.  Can you see anything in the history of the code in TFS to indicate when and why that happened?

**Comment by stephentoub** [comment]
> In particular

Also including a message like "Fixes githubissuenumber" so that when the PR is merged, GitHub automatically closes the associated issue.

I've made these changes for this PR.

**Comment by davidsh** [comment]
cc: @CIPop

**Comment by tijoytom** [comment]
Test Innerloop Windows_NT Debug Build and Test

**Comment by tijoytom** [comment]
@davidsh @stephentoub The test failure is un-related and the same test pass for Release.  Is it ok to merge?

**Comment by stephentoub** [comment]
> The test failure is un-related and the same test pass for Release. Is it ok to merge?

What was the test failure?  When you restart the leg, please include a mention of what the failure was.

And since you restarted it, let's let it finish before merging.

Thanks.

**Pull request merged**